filer/tests/admin.py

```diff
@@ -25,6 +26,12 @@
 from filer import settings as filer_settings
 
 
+def model_to_dict(instance, **kwargs):
+    if kwargs.pop('all'):
```

Comment:
I would do ``kwargs.pop('all', False)``. This way ``all`` is not required every single time.